app/routes/app+/rewards.addresses.tsx

```diff
@@ -58,8 +58,12 @@ export const loader = async (data: DataFunctionArgs) => {
     serviceProvider: user.address as `0x${string}`,
   });
   const networks = await listNetworks();
+  const userNetworks = wallets.map((w) => w.chain);
+  const newNetworks = networks.filter(
+    (network) => typeof userNetworks.find((un) => network.name === un.name) == "undefined"
```

Comment:
slight nitpick. Had some trouble parsing this so I tried to see if I could rewrite it:

```
const userNetworks = wallets.map((w) => w.chain.name);
const networks = await listNetworks();
const newNetworks = networks.filter((n) => !userNetworks.includes(n.name));
```